admin_app/src/app/content/components/DownloadModal.tsx

```diff
@@ -13,6 +13,7 @@ import Papa from "papaparse";
 import { useState } from "react";
 import { LoadingButton } from "@mui/lab";
 import { Content } from "../edit/page";
+import { Layout } from "@/components/Layout";
 
 const MAX_CARDS_TO_FETCH = 200;
 
```

Comment:
Apparently `ContentDownload` is declared on line 20 but never used. Can we delete it?

Reply:
It should have been used as the type for `list_json_content_download` - added now